wordpress-mobile/WordPress-iOS
# Feature/15104 today widget data

Ref #15104

This PR only prepares the `WordPressHomeWidgetToday` target to use the existing types used for the old Today widget. Also removes unnecessary additional entitlements file.

It does not (yet):
- Refresh data after the first load
- Allow to configure the site
- request fresh data from the backend
- Separate data from the old Today Widget

To test:

- Build run and install either size of the new Today widget
- You should see Today data from whatever site you had set for the old widgets (see example below)

<p align=center>
<img width=300 src="https://user-images.githubusercontent.com/34376330/97642755-f611e200-1a13-11eb-81ab-3adc268eb17b.png">
</p>

PR submission checklist:
